lib/active_model/serializer.rb

```diff
@@ -294,9 +294,7 @@ def initialize(object, options={})
 
       scope_name = @options[:scope_name]
       if scope_name && !respond_to?(scope_name)
-        self.class.class_eval do
-          define_method scope_name, lambda { scope }
-        end
+        self.singleton_class.send :alias_method, scope_name, :scope
       end
```

Comment:
good call.  In 0.10 I did  `define_singleton_method scope_name, lambda { scope }`, but I like alias_method